packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -236,6 +236,7 @@ export function getWebpackConfig(bundle: Bundle, bundleRefs: BundleRefs, worker:
       mainFields: ['browser', 'main'],
       alias: {
         core_app_image_assets: Path.resolve(worker.repoRoot, 'src/core/public/core_app/images'),
+        vega: require.resolve('vega/build/es5/vega.js'),
```

Comment:
@alexwizp I assume you see that this is not the right path, we need to pick the correct path based on the setup in the node_modules/vega directory